Content.Server/GameTicking/GameTicker.Spawning.cs

```diff
@@ -65,9 +65,17 @@ private void SpawnPlayers(List<IPlayerSession> readyPlayers, Dictionary<NetUserI
 
             // Calculate extended access for stations.
             var stationJobCounts = _stationSystem.Stations.ToDictionary(e => e, _ => 0);
-            foreach (var (_, (_, station)) in assignedJobs)
+            foreach (var (netUser, (_, station)) in assignedJobs)
             {
-                stationJobCounts[station] += 1;
+                if (station == EntityUid.Invalid)
```

Comment:
this is the tiniest ass nitpick, but this should check the job for nullability and not the station for being invalid.